openshift/openshift-virtualization.rst

```diff
@@ -178,7 +178,11 @@ Use the following steps to build the vGPU Manager container and push it to a pri
    * The NVIDIA vGPU Software is located on the **Driver downloads** tab of the **Software Downloads** page.
    * Click the **Download** link for the Linux KVM complete vGPU package.
      Confirm that the **Product Version** column shows the vGPU version to install.
-     Unzip the bundle to obtain the NVIDIA vGPU Manager for Linux (``NVIDIA-Linux-x86_64-<version>-vgpu-kvm.run`` file)
+     Unzip the bundle to obtain the NVIDIA vGPU Manager for Linux file, ``NVIDIA-Linux-x86_64-<version>-vgpu-kvm.run``.
+
+   .. include:: ../gpu-operator/gpu-operator-kubevirt.rst
```

Comment:
Some trickery.  I have a near-end-of-year resolution to set a better example for reusing information.